src/witan/gateway/components/query_router.clj

```diff
@@ -22,9 +22,9 @@
    :datastore/metadata-with-activities qds/metadata-with-activities
    :datastore/metadata-by-id qds/metadata-by-id
 
-   :search/dashboard search/dashboard
-   :search/datapack-files search/datapack-files
-   :search/datapack-files-expand search/datapack-files
+   :search/dashboard search/execute-search
+   :search/datapack-files search/execute-search
+   :search/datapack-files-expand search/execute-search
```

Comment:
If the gateway isn't differentiating between the search types I don't see any value in having these queries defined separately. `:search/query` ?

Reply:
It my understanding that these keys are used in the response routing. So, although they all call the same function, the results need to go to different places.